src/parsers/js/transformers/eslint1/index.js

```diff
@@ -0,0 +1,23 @@
+import {defineRule, runRule} from '../../utils/eslintUtils';
```

Comment:
The problem with this is that the transitive dependencies, `babel-eslint` and `babel-core` are now loaded on every page load, not only when someone uses the eslint transform.

You could simply load the utils dynamically as well.